src/ralph/ui/static/ui/css/rack.less

```diff
@@ -221,10 +221,12 @@
                     border-top: 1px solid lighten(@device-background, 10%);
                     background: @device-background;
                     &:hover {
-                        .active-border();
+                        border-bottom: 1px solid white;
```

Comment:
It was my mistake that active-border is a mixin. It should be a constant with value '1px solid white'. Then it could be used on top or bottom or around whole object appropriately.